Dockerfile.master.template

```diff
@@ -1,4 +1,4 @@
-FROM postgres:%%PG_MAJOR%%
+FROM debian:%%DEBIAN_VERSION%% as builder
```

Comment:
Question:
- It is possible to use - same base image as in the second stage ?
imho: it will be more secure;  

Extreme case:   when somebody building manually .. or just reuse this Docker script .. 
  -  postgres:13  pulled the latest .. but the base debian is not refreshed the same time.

Or there are a few day difference in the upstream "debian" and "postgres:13" ..
so the "postgres:13" base layer is older version ..

Reply:
Which is better, "postgres:12" or "postgres:13"?
I think Multistage build is useful for speed up building both 12-master and 13-master so I use `debian:buster-slim`.

Reply:
Ahh, Original comment is not focus the speed.
OK, I will change code.

Reply:
I change my code but this postgis hash may fail in test so I change this PR to Draft.
```
Run tests: 143
Failed: 1
make: *** [runtest.mk:10: check-regress] Error 1
```